scipy/stats/tests/test_distributions.py

```diff
@@ -1429,38 +1432,50 @@ def test_fit(self, rvs_shape, rvs_loc, rvs_scale):
                                 loc=(rvs_loc + 2))
         shape_mle_a, loc_mle_a, scale_mle_a = stats.pareto.fit(data, floc=2)
         assert_equal(scale_mle_a + 2, data.min())
-        assert_equal(shape_mle_a, 1/((1/len(data - 2)) *
-                                     np.sum(np.log((data
-                                                    - 2)/(data.min() - 2)))))
+
+        data_shift = data - 2
+        ndata = data_shift.shape[0]
+        assert_equal(shape_mle_a,
+                     ndata / np.sum(np.log(data_shift/data_shift.min())))
```

Comment:
This calculation previously introduced some instability as a result of taking the inverse at the end of the equation 
<img width="452" alt="Screen Shot 2022-01-29 at 12 43 59 PM" src="https://user-images.githubusercontent.com/44255917/151676969-e0ff02bf-f181-428a-92a7-2ba811e6f509.png">
.

Reply:
I understand now that because it is checking for exact equality this needs to be changed to match updates to the way this is calculated.